angular/jest.config.js

```diff
@@ -0,0 +1,18 @@
+const { pathsToModuleNameMapper } = require("ts-jest/utils");
+
+const { compilerOptions } = require("./tsconfig");
+
+module.exports = {
+  name: "angular",
+  displayName: "angular tests",
+  preset: "jest-preset-angular",
+  roots: ["<rootDir>/spec/"],
+  testMatch: ["**/+(*.)+(spec).+(ts)"],
+  setupFilesAfterEnv: ["<rootDir>/spec/test.ts"],
+  collectCoverage: true,
+  coverageReporters: ["html", "lcov"],
+  coverageDirectory: "coverage",
+  moduleNameMapper: pathsToModuleNameMapper(compilerOptions?.paths || {}, {
+    prefix: "<rootDir>/",
+  }),
```

Comment:
I tried it out and it just seems more confusing to me. We would have an actually common one, which would have the coverage stuff, a subprojects one containing `roots`, `testMatch`, and `setupFilesAfterEnv`, and then each project's file.

I think it would actually make it harder to write a new config.

Reply:
Fair enough.